.circleci/config.yml

```diff
@@ -111,3 +226,11 @@ workflows:
               executor: [openjdk8, openjdk11]
               # Run just the latest version. This saves resources trusts that the main test suite already exercises clojure compat sufficiently:
               clojure-version: ["1.10.3"]
+
+      - deploy:
+          requires:
+            - test
+            - test_third_party_deps
+          filters:
+            tags:
+              only: /Release-.*/
```

Comment:
We also need

```
            branches:
              ignore: /.*/
```

Reply:
I don't believe we do. Since this only runs for tags. This way you _can_ release from a branch, like I'm doing now.

Reply:
Else this will run in every build, which would fail for forks and potentially publish undesired releases?

Reply:
No. It will only run on tagged commits.

Reply:
I tried this branch on my personal CI account and it didn't seem so:

![image](https://user-images.githubusercontent.com/1162994/117135463-4002c880-ada7-11eb-8142-ee8a01b9c819.png)

the `deploy` job would run no matter what (and fail, given I lack the GPG env var)

Reply:
...The specific error being:

```
#!/bin/bash -eo pipefail
GNUPGHOME="$HOME/.gnupg"
export GNUPGHOME
mkdir -p "$GNUPGHOME"
chmod 0700 "$GNUPGHOME"

  echo "$GPG_KEY" \
   | base64 --decode --ignore-garbage \
   | gpg --batch --allow-secret-key-import --import

   gpg --keyid-format LONG --list-secret-keys

gpg: keybox '/home/circleci/.gnupg/pubring.kbx' created
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0

Exited with code exit status 2
CircleCI received exit code 2
```

As you hinted over Slack, perhaps we can simply detect if the remote is called `jonase`.

This is exposed as `CIRCLE_PROJECT_USERNAME`